### PR TITLE
Fixed #184

### LINKED ISSUE #184: GeometryAnimation crash
**Describe the bug**
when i click on morphing animation i get crash

**To Reproduce**
1. Go to Demo
2. Click on Morphing Animation

**Environment (please complete the following information):**
 - IDE vs2019

`System.FormatException: 'Unexpected token 'M41/416254,90C41/88876,82/5298 41/624584,57/2716 41/416254,50/99 36/29352,49/8907 36/894651,50/4264 31/567267,50/99 31/775597,44/59305 31/931627,42/43234 31/693536,36/77758 36/340008,36/79568 35/133525,36/76858 41/542523,36/58818 41/638383,32/81458 41/675605,33/50908 41/668792,29/20144 41/851382,25/46201 41/182183,18/65836 45/961941,14/23856 50/741699,9/81876 59/568752,10/59048 68/185297,10/70302 68/141127,16/44056 68/145937,20/8288 68/185297,23/58247 63/135373,24/4635 60/876109,23/24558 57/326157,24/84516 55/922779,28/231 55/756556,29/65047 56/63466,36/77758 59/21143,36/64422 64/265254,36/63978 68/690373,36/77758 67/695215,42/78657 67/742088,44/40134 66/922606,49/8464 63/245981,49/9344 61/316885,49/6976 55/810928,49/8464 55/662114,56/6024 56/168075,82/8571 55/810928,90 49/382352,90/1786 47/63649,89/97 41/416254,90z' encountered at position '2'.'
`
Untitled

### PR DESCRIPTION
**Pull request by ghost1372**
Finally I solved the problem 😁

### CODE REVIEW

File: src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs [hunk 1]
```diff
@@ -38,7 +38,15 @@ public string[] Strings
...
-        public GeometryAnimationUsingKeyFrames() => _areKeyTimesValid = true;
+        public GeometryAnimationUsingKeyFrames()
+        {
+            var currentCulture = System.Threading.Thread.CurrentThread.CurrentCulture.Parent.Name;
+            if (!currentCulture.Equals("en", StringComparison.OrdinalIgnoreCase))
+            {
+                System.Threading.Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo("en-US");
+            }
+            _areKeyTimesValid = true;
+        }
```
Comment (NaBian) [inline src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: So these codes means that as long as you don't use en, you default to en-us? I don't know if it's going to have any side effects while the #184 can be solved.

**Comment by ghost1372** [comment]
Yes
I think en-us is standard and I don't think it has any side effects 
The code works when the region settings in windows are not in English

File: src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs (hunk 1 above)
Comment (NaBian) [reply src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: ok, you put codes in GeometryAnimationUsingKeyFrames's ctor but what about GeometryAnimation?

Comment (ghost1372) [reply src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: It works well as you can see
However, I don't know if GeometryAnimation needs to change or not

Comment (NaBian) [reply src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: GeometryAnimationUsingKeyFrames was be used in demo, you can try GeometryAnimation too.

Comment (ghost1372) [reply src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: I tested the following code and it worked
 `System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;`
I think it's better than the previous code
I also added this code to GeometryAnimation

Comment (ghost1372) [reply src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs:45]: Hi again
I tested with the following code and it worked smoothly without `System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
` in GeometryAnimation

```
 <Storyboard x:Key="StoryboardOnLoaded2" RepeatBehavior="Forever" AutoReverse="True">
            <hc:GeometryAnimation From="{StaticResource FaceBookGeometry}" To="{DynamicResource TwitterGeometry}" Storyboard.TargetProperty="Data" Storyboard.TargetName="PathDemo2">
                <hc:GeometryAnimation.EasingFunction>
                    <QuarticEase EasingMode="EaseInOut"/>
                </hc:GeometryAnimation.EasingFunction>
                
            </hc:GeometryAnimation>
        </Storyboard>

            <Path Name="PathDemo2" Width="100" Height="100" Data="{StaticResource TwitterGeometry}" Fill="#3b5998"/>

```

**Pull request merged**